package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-persist-crosstab",
-  "version": "3.0.0",
+  "version": "3.5.0",
```

Comment:
BTW, this is assuming you want to keep the redux-persist-crosstab version in sync with the redux-persist version. Let me know if you'd prefer semver instead.